lib/parser.js

```diff
@@ -188,7 +198,20 @@ module.exports = function parser(identifier, string) {
   // Parses field options: [ name1 = value1, name2 = value2 ]
   function parseFieldOptions(parent) {
     parseBlock(function (token) {
+      var hasParens = false
+      var nameToken = token
+      if (token.type == Token.Type.START_PAREN) {
```

Comment:
I'm being blind, but what part of the grammar does this correspond to.  Maybe expand the example on line 198.